frontend/src/page/Realm.tsx

```diff
@@ -1,43 +1,65 @@
 import React from "react";
 import { Link } from "react-router-dom";
 
+import { graphql, useLazyLoadQuery } from "react-relay/hooks";
+import { RealmQuery } from "../query-types/RealmQuery.graphql";
+
 import { Breadcrumbs } from "../ui/Breadcrumbs";
 
 
 type Props = {
-    path: string[];
+    path: string;
```

Comment:
Mhhhh why? Or rather: how does it work without changing the routing stuff?

Reply:
I'm not sure what you mean. The router **gives** you a string. You just split it before to get an array. However, a string is actually what we want to call the API with.

Reply:
Oh oops, didn't remember I did the splitting.